torchgeo/datasets/pastis.py

```diff
@@ -0,0 +1,444 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""PASTIS dataset."""
+
+import abc
+import os
+from collections.abc import Sequence
+from typing import Callable, Optional, Union
+
+import fiona
+import matplotlib.pyplot as plt
+import numpy as np
+import torch
+from matplotlib.colors import ListedColormap
+from torch import Tensor
+
+from .geo import NonGeoDataset
+from .utils import check_integrity, download_url, extract_archive
+
+
+class PASTIS(NonGeoDataset, abc.ABC):
+    """PASTIS dataset.
+
+    The `PASTIS <https://github.com/VSainteuf/pastis-benchmark>`__
+    dataset is a dataset for time-series panoptic segmentation of agricultural parcels
+
+    Dataset features:
+
+    * support for the original PASTIS and PASTIS-R versions of the dataset
+    * 2,433 time-series with 10 m per pixel resolution (128x128 px)
+    * 18 crop categories, 1 background category, 1 void category
+    * semantic and instance annotations
+    * 3 Sentinel-1 Ascending bands
+    * 3 Sentinel-1 Descending bands
+    * 10 Sentinel-2 L2A multispectral bands
+
+    Dataset format:
+
+    * time-series and annotations are in numpy format (.npy)
+
+    Dataset classes:
+
+    0. Background
+    1. Meadow
```

Comment:
Could also do:
```
0. Background
#. Meadow
#. Soft Winter Wheat
#. ...
```
if you don't want to repeat the numbers, although this looks better when looking at the source code.

Reply:
Meh, I like the numbers